web/modules/weather_data/src/Service/HourlyForecastTrait.php

```diff
@@ -55,6 +55,32 @@ protected function getForecastPropertyAsHourlyPeriods($all, $propertyName)
         return $periods;
     }
 
+    public function filterHoursToSingleDay($hours)
+    {
+        $times = array_column($hours, "time");
+        $firstTime = explode(" ", $times[0]);
+
+        $first6am = array_search("6 AM", $times);
+        if ($first6am === 0) {
+            $first6am = array_search("6 AM", array_slice($times, 1));
+        }
+
+        if ($firstTime[1] === "AM" && $firstTime[0] < 6) {
```

Comment:
Will PHP implicitly convert the string "6" to an int? If not we might want to do it explicitly here so the comparison doesn't somehow blow up right?

Reply:
It does implicitly cast for the comparison, but you're right, we should be explicit anyway. Change incoming!